docs/RELEASE_NOTES.md

```diff
@@ -1,12 +1,13 @@
 # MongoDB Kubernetes Operator 0.6.2
 
 ## Kubernetes Operator
-  
+
 - Changes
   - stability improvements when changing version of MongoDB.
   - increased number of concurrent resources the operator can act on.
   - mongodb will now send its log to stdout by default.
   - changed the default values for `MONGODB_REPO_URL` and `MONGODB_IMAGE` in the operator deployment
+  - added the support of arbiters for the replica sets.
```

Comment:
Can you merge master and replace the release notes? We just made a release so this line you added should be the only entry in the 0.6.3 version

Reply:
Yes no problem